lib/ui-utils.js

```diff
@@ -9,6 +9,8 @@ class UiUtils {
 
 }
 
+UiUtils.prototype.$ = require('jquery');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier name '$' is too short. (< 3)](https://www.codacy.com/app/connor-bray/atom-maven/file/2738198029/issues/source?bid=3352650&fileBranchId=3353288#l12)